<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_soan_de_thi</class>
 <widget class="QWidget" name="man_hinh_soan_de_thi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="2,5,0" columnstretch="1,3">
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 112, 192);
border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QComboBox" name="mon_hoc">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>55</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
background-color: rgb(255,255,255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Môn học</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QComboBox" name="hoc_ky">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>55</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
background-color: rgb(255,255,255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Học kỳ</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QComboBox" name="nam_hoc">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
background-color: rgb(255,255,255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>35</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Năm học</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="sua">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Sửa</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/334_n.png</normalon>
          </iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="thoat">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Thoát</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/32_n.png</normalon>
          </iconset>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 112, 192);
border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>77</y>
        <width>672</width>
        <height>56</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="tieu_de">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>15</y>
         <width>541</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tiêu đề</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>13</y>
        <width>281</width>
        <height>57</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ngày thi</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="ngay_thi">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
</string>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>314</x>
        <y>13</y>
        <width>371</width>
        <height>57</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="thoi_luong">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Thời lượng:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>phút</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 112, 192);
border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTableWidget" name="danh_sach_cau_hoi">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>14</y>
        <width>671</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(143, 143, 143)</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>310</y>
        <width>670</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="xoa_cau_hoi">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
background-color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>➖</string>
       </property>
      </widget>
      <widget class="QPushButton" name="them_cau_hoi">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
background-color: rgb(85, 255, 127);</string>
       </property>
       <property name="text">
        <string>➕</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>5</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Số câu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="so_cau">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>5</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(0, 0, 0);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
